py/desisurvey/rules.py

```diff
@@ -144,16 +147,18 @@ def __init__(self, file_name='rules.yaml', restore=None):
 
             # Calculate priority multipliers to implement optional DEC ordering.
             dec_order = node.get('dec_order')
-            if dec_order is not None:
+            if dec_order is not None and np.any(group_sel):
                 dec_group = dec[group_sel]
                 lo, hi = np.min(dec_group), np.max(dec_group)
+
                 slope = float(dec_order)
+                epsilon = float(hi>lo)  #- used to avoid 0.0 / 0.0
```

Comment:
Shouldn't this be `float(lo>=hi)` so epsilon is normally zero?  In any case, epsilon=0 or 1 doesn't seem intuitive here.  Perhaps simpler to just add a third if clause below?
```
if lo == hi:
  dec_priority[group_sel] = 1.
elif slope > 0:
  ...
else:
  ...
```